src/optimus/optimizations/minify.clj

```diff
@@ -1,7 +1,8 @@
 (ns optimus.optimizations.minify
   (:require
     [clojure.string :as str]
-    [optimus.js :as js]))
+    [optimus.js :as js]
+    [clojure.java.io :as jio]))
```

Comment:
Nitpick: Please change the alias for `clojure.java.io` to `io` since that's used elsewhere in the code base :)